server/engine/src/main/java/org/eclipse/lsp/cobol/service/copybooks/CopybookIdentificationCombinedStrategy.java

```diff
@@ -47,12 +47,19 @@ public CopybookIdentificationCombinedStrategy(
    * @return True if it's a copybook. False otherwise
    */
   @Override
-  public boolean isCopybook(String uri, String text, List<String> config)
-      throws UndeterminedDocumentException {
+  public boolean isCopybook(String uri, String text, List<String> config) {
     try {
-      return service1.isCopybook(uri, text, config);
+      if (!zoweExplorerFile(uri)) {
+        return suffixStrategy.isCopybook(uri, text, config);
+      } else {
+        return contentStrategy.isCopybook(uri, text, config);
+      }
     } catch (UndeterminedDocumentException ex) {
-      return service2.isCopybook(uri, text, config);
+      return contentStrategy.isCopybook(uri, text, config);
     }
   }
+
+  private static boolean zoweExplorerFile(String uri) {
+    return uri.contains("/zowe.vscode-extension-for-zowe");
```

Comment:
This is not reliable because the use can specify the location of downloaded files through `zowe.files.temporaryDownloadsFolder.path` setting.